src/fiatconnect/ReviewScreen.tsx

```diff
@@ -65,6 +64,11 @@ export default function FiatConnectReviewScreen({ route, navigation }: Props) {
   const feeType = FeeType.SEND
   const tokenList: TokenBalance[] = useSelector(tokensListWithAddressSelector)
   const cryptoType = normalizedQuote.getCryptoTypeString()
+  // TODO: remove deprecated function call when FiatConnect updated to take token IDs
```

Comment:
Let's have a placeholder issue and link them here and other places so this isn't a TODO we forget.